client/jetpack-cloud/sections/pricing/header/index.tsx

```diff
@@ -7,33 +7,41 @@ import { translate } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import FormattedHeader from 'components/formatted-header';
-import { preventWidows } from 'lib/formatting';
+import { isEnabled } from 'calypso/config';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { preventWidows } from 'calypso/lib/formatting';
 import JetpackComMasterbar from '../jpcom-masterbar';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const Header = () => (
-	<>
-		<JetpackComMasterbar />
-		<div className="header">
-			<FormattedHeader
-				className="header__main-title"
-				headerText={ preventWidows(
-					translate( 'Security, performance, and marketing tools for WordPress' )
-				) }
-				align="center"
-			/>
-			<p>
-				{ translate(
-					'Get everything your site needs, in one package — so you can focus on your business.'
+const Header = () => {
+	const isAlternateSelector = isEnabled( 'plans/alternate-selector' );
+	const header = isAlternateSelector
+		? translate( 'Security, performance, and growth tools for WordPress' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Complete WordPress security, performance, and growth' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/3-col-plans-remove-option-cards)